validation/gas_limit_test.go

```diff
@@ -0,0 +1,62 @@
+package validation
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func testGasLimit(t *testing.T, chain *superchain.ChainConfig) {
+	rpcEndpoint := superchain.Superchains[chain.Superchain].Config.L1.PublicRPC
+	require.NotEmpty(t, rpcEndpoint)
+
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	err = CheckGasLimit(chain, client, nil)
+	require.NoError(t, err)
+}
```

Comment:
Wondering if this should live in `gas_limit.go` or not? Don't feel strongly but it might be nice to clearly separate the new unit tests from the hook for the existing validation suite.

Reply:
I don't feel strongly either. I changed it in https://github.com/ethereum-optimism/superchain-registry/pull/802/commits/1e73866041c407e6399bdb63ea5613b901a3513d. Will revert that change